.github/workflows/main.yaml

```diff
@@ -55,25 +55,26 @@ jobs:
       # git for cache keys.
       - name: Generate cache key
         run: |
-          echo $(git rev-parse HEAD:App) > ./dd-cache-key.txt
-          echo $(git rev-parse HEAD:ios) >> ./dd-cache-key.txt
+          echo $(git rev-parse HEAD:core/App) > ./dd-cache-key.txt
```

Comment:
There isn't anything useful in core/ to cache here. It's all in `ios`.

Reply:
I was just maintaining compatibility with what already was there. Can this be addressed later?

Reply:
Yup